.meteor/packages

```diff
@@ -73,3 +73,4 @@ perak:markdown
 seriousm:emoji-continued
 templates:tabs
 verron:autosize
+pfafman:filesaver
```

Comment:
As discussed in https://github.com/wekan/wekan/issues/396#issuecomment-164301574 I would favor a server-side export generation over a client one.